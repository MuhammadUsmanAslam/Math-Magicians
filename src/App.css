.app {
  width: 100vw;
  height: auto;
  min-height: 100vh;
  display: flex;
  align-items: center;
  flex-direction: column;
  background-color: rgb(248, 235, 215);
}

ul {
  list-style: none;
}

a {
  text-decoration: none;
}
