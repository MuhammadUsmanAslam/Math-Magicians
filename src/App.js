import './App.css';
import Calculator from './components/Calculator';
import Quotes from './components/Quotes.js';

function App() {
  return (
    <div className="app">
      <Calculator/>
      <Quotes/>
    </div>
  );
}

export default App;
