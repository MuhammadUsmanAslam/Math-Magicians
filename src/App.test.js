import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
import App from "./App";

test("render App component", () => {
	const tree = renderer
		.create(
			<Router>
				<App></App>
			</Router>
		)
		.toJSON();

	expect(tree).toMatchSnapshot();
});
