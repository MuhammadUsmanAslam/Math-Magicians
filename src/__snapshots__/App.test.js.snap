// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App component 1`] = `
<div
  className="app"
>
  <div
    className="header"
  >
    <a
      className="header__logo"
      href="/"
      onClick={[Function]}
    >
      Math Magicians
    </a>
    <ul
      className="header__nav-links"
    >
      <li>
        <a
          className=""
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className=""
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          className=""
          href="/quotes"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </div>
  <div
    className="home"
  >
    <h1
      className="home__heading"
    >
      Welcome to our page!
    </h1>
    <p
      className="home__para"
    >
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quasi, distinctio. Dolorem unde distinctio reprehenderit ducimus blanditiis magnam deserunt, soluta, voluptatibus tenetur saepe deleniti aut. Aut odit cupiditate ab aliquam non.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quasi, distinctio. Dolorem unde distinctio reprehenderit ducimus blanditiis magnam deserunt, soluta, voluptatibus tenetur saepe deleniti aut. Aut odit cupiditate ab aliquam non.
    </p>
    <p
      className="home__para"
    >
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quasi, distinctio. Dolorem unde distinctio reprehenderit ducimus blanditiis magnam deserunt, soluta, voluptatibus tenetur saepe deleniti aut. Aut odit cupiditate ab aliquam non.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quasi, distinctio. Dolorem unde distinctio reprehenderit ducimus blanditiis magnam deserunt, soluta, voluptatibus tenetur saepe deleniti aut. Aut odit cupiditate ab aliquam non.
    </p>
  </div>
</div>
`;
